mast_aladin_lite/mast_overlay.py

```diff
@@ -0,0 +1,37 @@
+from enum import Enum
```

Comment:
question (nit):  any thoughts about moving these overlay files into an overlay subdirectory to help with organization?